tests/ExpressionTest.php

```diff
@@ -258,6 +258,26 @@ public function testNestedExpressions()
     }
 
     /**
+     * @covers ::__toString
+     * @expectedException \LogicException
+     */
+    public function testToStringException1()
```

Comment:
this triggers crash in test-suite, but because it produced "zero" output it wasn't failed.